http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```
@@ -594,12 +599,15 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       )
       app <- appQuery.save(saveApp, Some(ctx.traceId)).transaction
 
+      landingZoneResources <- getLandingZoneResources(workspaceDesc.spendProfile, userToken)
```

Thinking we can move this up, closer to the other WSM call on L479?

That way if we fail to get the LZ we can fail faster, before inserting app records to the DB.

---

Also -- maybe we should only do this if the cloud provider is Azure. This would mean making `LandingZoneResources` optional in the pubsub message.

---

Oh, oops! Moved and added checking for cloud provider-